test/sql/updates/post.continuous_aggs.v2.sql

```diff
@@ -16,6 +16,10 @@ SELECT view_name, schedule_interval, materialized_only, materialization_hypertab
 
 SELECT maxtemp FROM mat_ignoreinval ORDER BY 1;
 
+-- Note that greatest_modified value for tables with an 'ignore_validation_other_than' won't match
```

Comment:
Could you elaborate on this comment?

Reply:
I was trying to improve this check a bit, but it turned out to not be possible, so that change and this comment have been removed.